sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/GrpcServiceBuilder.java

```diff
@@ -25,6 +25,9 @@
   GrpcServiceBuilder<ReqMarshalerT, ResUnMarshalerT> setTrustedCertificates(
       byte[] trustedCertificatesPem);
 
+  GrpcServiceBuilder<ReqMarshalerT, ResUnMarshalerT> setClientTls(byte[] privateKeyPem,
+      byte[] privateKeyChainPem);
```

Comment:
Have people asked for client cert support for jaeger remote sampling?

Reply:
I have tried to add client cert everywhere where server certificate could be used. 
Is there any problem with having this?

Reply:
Just slightly more surface area to maintain. I don't feel strongly about this thought.